apps/camera/style/recording-timer.css

```diff
@@ -58,6 +63,11 @@
   transform: rotate(-270deg);
 }
 
+[dir=rtl] .deg270 .recording-timer {
+  right: -5rem;
+  left: auto;
+}
+
```

Comment:
Fixes recording timer 50px shift when phone is rotated 90 and 270 deg in RTL mode